contributed_definitions/NXfabrication.nxdl.xml

```diff
@@ -25,6 +25,12 @@
     <doc>
          Details about a component as it is defined by its manufacturer.
     </doc>
+    <field name="name" type="NX_CHAR">
```

Comment:
what does it differ from model?

Reply:
Model is the version of the component or device, do not say what device it is. It might be an "amplifier"(`name field`) that might have several "versions"(`model field`) with different scales of capabilities.

Reply:
Model is for the name of the model and if there are different versions, they can got to the new attribute model@version, so 'name' is not needed.

Reply:
Right! Device name or component name is not part of fabrication, therefore, the name is ambiguous with model.

Reply:
The field `name` has also been removed, please take a look at the file for the last commit.